### PR TITLE
Fix error rendering multiple Google Maps on the same page

### PR DESCRIPTION
**Pull request by CodyReichert**
I wanted to PR this fix for you since I was already pushing it to SimplyRETS/simplyretswp :)

This _should_ fix the issue loading multiple maps on the same page. See all the comments on the changes here: https://github.com/SimplyRETS/simplyretswp/pull/98

One question: are you using [sr_listings] _and_ [sr_map_search] on the same page? IIRC, you were just using the former. This change still doesn't allow both of those to be mingled on the same page (I can fix that), but it does allow each of the to be used multiple times on the same page.

In other words, this will work:
```
[sr_listings]
[sr_listings]
```
But this will not:
```
[sr_listings]
[sr_map_search]
```

If your use-case was the second one, let me know I'll investigate a fix for that one!

### CODE REVIEW

**Comment by catedm** [comment]
I am using [sr_listings limit="3"] twice on the page. So hopefully this patch will fix it. Thanks!

**Comment by CodyReichert** [comment]
:ok_hand: Just released the official one, in case you need it for anything. Let me know if you're still hitting any issues!
